src/system.c

```diff
@@ -238,13 +238,14 @@ void cw_system_cpu_cycle(void)
                     {
                         chip8.register_v[0xF] = false;
                         uint8_t overflow = false;
+                        uint8_t result = chip8.register_v[INSTR_POS_X] - chip8.register_v[INSTR_POS_Y];
 
-                        if (chip8.register_v[INSTR_POS_X] >= chip8.register_v[INSTR_POS_Y]) {
+                        if (chip8.register_v[INSTR_POS_X] > chip8.register_v[INSTR_POS_Y]) {
```

Comment:
I think this one needs to be the other way around
```suggestion
                        if (chip8.register_v[INSTR_POS_Y] > chip8.register_v[INSTR_POS_X]) {
```